keyboards/yandrstudio/keit75/info.json

```diff
@@ -0,0 +1,43 @@
+{
+    "keyboard_name": "keit75",
```

Comment:
```suggestion
    "keyboard_name": "keit75",
    "manufacturer": "QMK",
```

Reply:
All "maintainer", "manufacturer" and "vid" information of have been inherited from file "https://github.com/qmk/qmk_firmware/blob/master/keyboards/yandrstudio/info.json" in the root directory.